pycparser/_build_tables.py

```diff
@@ -20,7 +20,7 @@
 # Generate c_ast.py
 from _ast_gen import ASTCodeGenerator
 ast_gen = ASTCodeGenerator('_c_ast.cfg')
-ast_gen.generate(open('c_ast.py', 'w'))
+ast_gen.generate(open('c_ast.py', 'w', encoding='UTF-8')))
```

Comment:
Why is this change in this PR?

Reply:
it raises the following error when building without that part:
`UnicodeEncodeError: 'charmap' codec can't encode characters in position 2511-2512: character maps to <undefined>`
Those characters end up corresponding to '└' and '─'. No idea why as it should default to UTF-8. but adding the encoding specifier fixes it.